doc/api/number/number-formatter.md

```diff
@@ -122,6 +122,28 @@ formatter( 0.00012345 );
 // > "0.000123"
 ```
 
+Format percentages:
```

Comment:
This deserves some formatting as a header, but since the other labels have the same issues, that should be dealt with separately. Along with changing the "Example" header to plural "Examples".